src/EnergyPlus/WaterUse.cc

```diff
@@ -1658,7 +1657,7 @@ namespace WaterUse {
         // PURPOSE OF THIS SUBROUTINE:
         // Calculates the zone internal gains due to water use sensible and latent loads.
 
-        bool MyEnvrnFlagLocal(true);
+        static bool MyEnvrnFlagLocal = true;
```

Comment:
[src/EnergyPlus/WaterUse.cc:1665]:(style),[knownConditionTrueFalse],Condition 'MyEnvrnFlagLocal' is always true